drivers/counter/counter_nrfx_rtc.c

```diff
@@ -184,7 +199,12 @@ static void event_handler(nrfx_rtc_int_type_t int_type, void *p_context)
 
 	if (int_type == COUNTER_TOP_INT) {
 		/* Manually reset counter if top value is different than max. */
-		if (data->top != COUNTER_MAX_TOP_VALUE) {
+		if ((data->top != COUNTER_MAX_TOP_VALUE)
+#if IS_PPI_WRAP_ENABLED()
+		    && !((const struct counter_nrfx_config *)
```

Comment:
What's the point in casting `bool` to this pointer?

Reply:
cleaned it. casting was not needed.